extensions/binding/org.eclipse.smarthome.binding.sonyaudio/OSGI-INF/SonyAudioDiscovery.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you switch to using DS annotations? (check other bindings or the latest archetype on how that looks like).